Package.swift

```diff
@@ -27,16 +27,48 @@
 
 import PackageDescription
 
-let package = Package(
+public let package = Package(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [public declarations should be documented.](https://app.codacy.com/app/FabrizioBrancati/SwiftyBot/pullRequest?prid=3796373)